articles/site-recovery/avs-tutorial-dr-drill-azure.md

```diff
@@ -39,7 +39,7 @@ Complete the previous tutorials:
 
 ## Verify VM properties
 
-Before you run a test failover, verify the VM properties, and make sure that the  [VMware VM](vmware-physical-azure-support-matrix.md#replicated-machines) complies with Azure requirements.
+Before you run a test failover, verify the VM properties, and make sure that the  [VMware vSphere VM](vmware-physical-azure-support-matrix.md#replicated-machines) complies with Azure requirements.
```

Comment:
Please use "VMware VMs".

Reply:
@Sharmistha-Rai - unfortunately VMware VMs is a misnomer and we need to follow the naming guidelines of VMware. VMware vSphere is VMware product that runs Virtual Machines